bgpd/bgp_route.c

```diff
@@ -10829,45 +10845,41 @@ DEFUN(show_ip_bgp, show_ip_bgp_cmd,
 }
 
 /* BGP route print out function with JSON */
-DEFUN (show_ip_bgp_json,
-       show_ip_bgp_json_cmd,
-       "show [ip] bgp [<view|vrf> VIEWVRFNAME] ["BGP_AFI_CMD_STR" ["BGP_SAFI_WITH_LABEL_CMD_STR"]]\
+DEFUN(show_ip_bgp_json, show_ip_bgp_json_cmd,
```

Comment:
can you leave the previous version of DEFUN? I mean just add a json + description.